mentoring/processes/group-mentoring.md

```diff
@@ -1,23 +1,75 @@
-mentees should not be surprised by the outcome of the program.
+# Running a Contributor Ladder Growth Program 
+
+There are differences in ciricuulum for reviewer and Chair and/or Tech Lead 
+groups but the foundation of each are mostly the same unless otherwise noted 
+below in the set up instructions. This difference is also reflected in our 
+[community membership guidelines].
+
+## Discovery of Needs
+
+Cadence:  
+ContribEx sets a quarterly issue in k/community requesting SIGs to participate 
+
+Rolling:  
+SIGs can approach ContribEx at anytime to start a group; file an issue in 
+kubernetes/community or talk with us in #sig-contribex  
+
+## Ciriculum Building
+While not a fully structured program, some light planning is needed to make sure
+the ciriculum being taught is relevant for the group. Here are our base 
+suggestions:
+  
+*Reviewer*  
+Enhancements
+Development
+- Triage
+- Reviews
+- API Changes
+- Testing
+Code Freeze
+Cherry-Picks/Branch Management
+
+*Chair*
+KEPs++ 
+Meetings and Organization
+Responsbilities and governance overview
+Consensus-building
+Communicating in our big world  
+
+*Chair/TL combo*
+Chair + Reviewer
+[add more here]
+
+One of the best strategies is to have everyone work on something together, 
+examples:
+- reviewing guidelines for the SIG if there are none/updating guide
+- CONTRIBUTING.md for the SIG if there isn't one/updating guide
+- mid-size project
+- a release cycle of features for the SIG, establishing better processes
+
+## Setup and Outreach  
+0. Need is established and SIG provides a mentor*
+1. ContribEx kicks off two issues, sets a target start and end date, and selects
+a coordinator
+  - issue #1: outreach issue to collect folks who are interested, issue will 
+  close when there are the max participants needed
+  - issue #2 setting up infra - template here: [TODO link to example]
+2. ContribEx or SIG member posts the issue to the following places:
+  - SIG and k-dev mailing lists
+  - #diversity, #kubernetes-contributors, #sig-[name], and other slack channels
+3. Coordinator arranges a doodle for the kick off meeting and days/times that 
+would be best for the mentor and group for slack standups and touch points. Once conscensus is reached, coordinator schedules it.
+4. Mentor creates template that serves as a reference guide for the group. See
+a sig-node example here: https://hackmd.io/8i8prErgSOamUmEZDDZPWA
+5. Program begins at the kick off meeting
+  - [TODO add kick off template]
+6. Feedback is continuous as you stand up weekly; however, Coorindator sets up
+a mid program check-in. In this check-in, members of the group will start their 
+application process for their roles. 
+  - [TODO add community membership template]
+7. By this point, folks should know where they stand. At the end of the program,
+ mentors and/or individuals will start the PR process in OWNERS files for those 
+who are ready  
```

Comment:
For Chair/TL roles, there aren't really OWNERS files to update in order to kick off a transition. This might be pointing to a gap in SIG leadership succession processes.

Reply:
there is sigs.yaml